src/uniforms/RenderTarget2DArray.js

```diff
@@ -54,12 +55,22 @@ export class RenderTarget2DArray extends WebGLArrayRenderTarget {
 		for ( let i = 0, l = depth; i < l; i ++ ) {
 
 			const texture = textures[ i ];
+			//save texture uv transform
+			texture.updateMatrix();
+			const uvTranform = texture.matrix;
+			texture.matrixAutoUpdate = false;
+			texture.matrix = new Matrix3();
```

Comment:
We should avoid creating new objects as much as possible to prevent garbage creation. It would be best to just call `.identity()` to modify in place. Setting `matrixAutoUpdate` to false should be good enough otherwise